### DISCUSSION TITLE
Support for version 4?

### DISCUSSION THREAD

**Discussion by JonBensi** [spulec/moto#7375] [General]
Will there be any more patches for version 4? I've had a few vulnerability alerts for various packages that are used for some of the extras used in version 4 of Moto. If not that is fine, but was just curious if there was a desire to keep 4 going or to have everyone move to 5? Thanks!

**Comment by bblommers** [comment]
Hi @JonBensi! No, I don't intend to make any v4 releases anymore. I know V5 has some major changes and might take some time to upgrade to, but I don't have any desire to manage multiple release/maintenance streams.
